contracts/peaks/yearn/YVaultPeak.sol

```diff
@@ -116,13 +116,24 @@ contract YVaultPeak is OwnableProxy, Initializable, IPeak {
         return inAmount.mul(yUSDToUsd()).div(1e18);
     }
 
+    // Bug - convert peak yCRV into yUSD on redeem
     function redeemInYusd(uint dusdAmount, uint minOut) external {
         core.redeem(dusdAmount, msg.sender);
         uint r = dusdAmount.mul(1e18).div(yUSDToUsd()).mul(redeemMultiplier).div(MAX);
         // there should be no reason that this contracts has yUSD, however being safe doesn't hurt
         uint b = yUSD.balanceOf(address(this));
         if (b < r) {
             controller.withdraw(yUSD, r.sub(b));
+            uint peak_yUSD = yUSD.balanceOf(address(this));
```

Comment:
That's `b`. Also please use camel case. so can consider renaming `b` to `peakYusd`.